Tests/iaas/flavor-naming/flavor-names-openstack.py

```diff
@@ -187,28 +211,42 @@ def main(argv):
     # Produce dicts for YAML reporting
     flvSCSList = {
         "MandatoryFlavorsPresent": MSCSFlv,
-        "MandatoryFlavorsMissing": scsMandatory,
+        "MandatoryFlavorsMissing": scsMandatory
```

Comment:
I would strongly advocate for trailing commas here and below!

Reply:
No strong opinion. I tend to think it's a good habit to have these commas, so I just did it.
Implemented in https://github.com/SovereignCloudStack/standards/pull/294/commits/84fde967cbb3119ab87667a7bb5569b39b0d8269